deploy/docker-compose.yml

```diff
@@ -1,21 +1,27 @@
 version: '3.2'
 services:
   datameta-app:
-    image: "lkuchenb/datameta:dev"
+    image: "datameta/datameta:dev"
     environment:
       SQLALCHEMY_URL: postgresql://datameta:datameta@datameta-postgresql/datameta
       SESSION_URL: datameta-memcached:11211
       SESSION_KEY: datameta
-      SESSION_SECRET: changeme
+      SESSION_SECRET: cheeph9shiew8baNg0iupohzuboi9efiuqu4ae2chien1naeWeew0uuviiSh8dee
       DATAMETA_STORAGE_PATH: /var/datameta/storage
       DATAMETA_DEMO_MODE: "true"
-      LETSENCRYPT_HOST:
-      VIRTUAL_HOST:
+      DATAMETA_INITIAL_FULLNAME:
+      DATAMETA_INITIAL_EMAIL:
+      DATAMETA_INITIAL_PASS:
+      DATAMETA_INITIAL_GROUPNAME:
+      LETSENCRYPT_HOST:   # Example: "dev.cogdat.de"
+      VIRTUAL_HOST:       # Example: "dev.cogdat.de"
```

Comment:
```suggestion
      LETSENCRYPT_HOST:   # Picked up by nginx-proxy / letsencrypt. Comma separated list of hostnames.
      VIRTUAL_HOST:       # Picked up by nginx-proxy / letsencrypt. Comma separated list of hostnames. Every hostname in LETSENCRYPT_HOST must appear also here, otherwise cert acquisition will fail
```